src/include/txn_inline.h

```diff
@@ -476,7 +476,8 @@ __wt_txn_oldest_id(WT_SESSION_IMPL *session)
      */
     WT_READ_BARRIER();
 
-    if (!F_ISSET(conn, WT_CONN_RECOVERING)) {
```

Comment:
Can you add a comment explaining the reason why the logged tables are not required to consider the recovered checkpoint snapshot as a minimum? The logged tables are not rollbacked as part of RTS, so there is no point in holding their oldest_id.

Reply:
Done

Reply:
`rollbacked` is not a proper word I think. How about something like the following?
`The logged tables are excluded as part of RTS, so there is no need of holding their oldest_id`?

Reply:
This makes more sense. Thanks.